src/main/app/src/components/SignUp.js

```diff
@@ -0,0 +1,100 @@
+import { Button, Grid, TextField, makeStyles, Paper, useMediaQuery, useTheme } from '@material-ui/core';
+import axios from 'axios';
+import React, { useState } from 'react';
+
+
+//TODO Make Inline Styling
+
+const useStyles = makeStyles(theme => ({
+  divStyle: {
+    alignContent: 'centre',
+    display: 'flex',
+    flexDirection: 'column',
+    maxWidth: 400,
+    minWidth: 30,
+    borderRadius: 20,
+    border: '30px solid',
+    borderImage: 'linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)',
+    padding: 50,
+    borderImageSlice: 1
+  }
+}));
+
+const postMe = async data => {
+  try {
+    const api = '/api/signup/';
+    const posted = await axios.post(api, data);
+    console.log(`STATUS CODE: ${posted.status}`);
+    console.log(`DATA: ${posted.data || "Nothing"}`);
+    return posted.status == 200 ? posted.data || {} : {};
+  } catch (err) {
+    console.log(err);
+    return err;
+  }
+};
+
+
+const SignUpForm = () => {
+  const classes = useStyles();
+  const [userName, setUserName] = useState("");
+  const [password, setPassword] = useState("");
+  const [email, setEmail] = useState("");
+  const signup = () => {
+    var sendData = new URLSearchParams();
```

Comment:
Use `let` or `const` instead of `var` - I would recommend just not even bothering with `var` in javascript, `let` and `const` are the more modern replacements